.github/workflows/daily.yaml

```diff
@@ -68,74 +68,132 @@ jobs:
+            # Get the container image name that was parsed out of the Dockerfile for the website.
+            container=$(cat website/docs/charts/description_list.md | grep "\[${chartname}\]" | cut -f3 -d '|' | grep -v 'Not Found' || echo "")
+            # Convert the container image name to a URL.
+            if [ ! -z "$container" ]; then
+              prefix=""
+              case "$container" in
+                lscr.io/linuxserver/*)
+                  prefix="https://fleet.linuxserver.io/image?name="
+                  container=${container#lscr.io/}
+                  ;;
+                tccr.io/truecharts/*)
+                  prefix="https://github.com/truecharts/containers/tree/master/mirror"
+                  container=${container#tccr.io/truecharts/}
+                  ;;
+                mcr.microsoft.com/*)
+                  prefix=""
+                  ;;
+                public.ecr.aws/*)
+                  prefix="https://gallery.ecr.aws/"
+                  container=${container#public.ecr.aws/}
+                  ;;
+                ghcr.io/*)
+                  prefix="https://"
+                  ;;
+                quay.io/*)
+                  prefix="https://"
+                  ;;
+                gcr.io/*)
+                  prefix="https://"
+                  ;;
+                *.azurecr.io/*)
+                  prefix=""
+                  ;;
+                *.ocir.io/*)
+                  prefix=""
+                  ;;
+                # There have been a number of domains used for the Docker Hub registry over the years.
+                # NOTE:  This is also the default case!
+                docker.io/*|index.docker.io/*|registry-1.docker.io/*|registry.hub.docker.com/*|*)
+                  prefix="https://hub.docker.com/r/"
+                  container=${container#docker.io/}
+                  container=${container#index.docker.io/}
+                  container=${container#registry-1.docker.io/}
+                  container=${container#registry.hub.docker.com/}
+                  # If the image name does not contain a slash it is a Docker Official Image.
+                  if [ "$container" == "${container////}" ]; then
+                    prefix="https://hub.docker.com/_/"
+                  # If the user name is library it is a Docker Official Image.
+                  elif [ "${container%%/*}" == "library" ]; then
+                    prefix="https://hub.docker.com/_/"
+                    container=${container#library/}
+                  fi
+                  # Avoid creating a bad link since an unsupported registry may have been used.
+                  slashes=${container//[^\/]/}
+                  # Bail out if the image name has more than 1 slash.
+                  if [ ${#slashes} -gt 1 ]; then
+                    prefix=""
+                    echo "WARNING: Not assuming '$container' is a Docker Hub image"
```

Comment:
I introduced printing this warning but didn't redirect it to stderr as I wasn't sure of how such things should be handled, but it seemed appropriate feedback to generate.  But maybe it belongs somewhere else, like where the Dockerfile gets added to the containers repository instead?

Reply:
I think it's fine to warn about this here :)